schema/geometry_schema.json

```diff
@@ -444,6 +453,23 @@
 							"vertex_id"
 						]
 					}
+				},
+				"floor_to_ceiling_height_override": {
```

Comment:
probably need plenum height overrides too right @shorowit ?

Reply:
I'm not crazy about adding override to the name, I tend to just imply that values set at lower levels override values from higher levels